### PR TITLE
Update try it online in the documentation.

### PR DESCRIPTION
**Pull request by HaudinFlorence**
In the documentation, in index.rst, change the try it online : replace the retrolite directive by 2 links to retro and lab.

### CODE REVIEW

File: docs/source/_contents/ipyleaflet.ipynb [hunk 1]
```diff
@@ -0,0 +1,53 @@
+{
```
Comment (martinRenou) [inline docs/source/_contents/ipyleaflet.ipynb:1]: I guess you did not want to commit this file. We should probably add `docs/source/_contents` to the `.gitignore`

File: docs/source/ipyleaflet.ipynb [hunk 2]
```diff
@@ -1,16 +1,5 @@
...
-   "cell_type": "code",
-   "execution_count": null,
-   "id": "8eee1995-a52b-4eae-8898-047746e57b19",
-   "metadata": {},
-   "outputs": [],
-   "source": [
-    "import piplite\n",
-    "await piplite.install('ipyleaflet')"
-   ]
-  },
```
Comment (martinRenou) [inline docs/source/ipyleaflet.ipynb:13]: We probably need to keep this cell around?

**Review by martinRenou — CHANGES_REQUESTED** [review]

File: docs/source/index.rst [hunk 3]
```diff
@@ -5,9 +5,8 @@ ipyleaflet: Interactive maps in the Jupyter notebook
 Try it online
 -------------
 
-You can try ipyleaflet directly in this documentation page thanks to JupyterLite!
-
-.. retrolite:: ipyleaflet.ipynb
+Directly from this page, you can make a try with ipyleaflet with clicking on `Try it on RetroLab <https://ipyleaflet.readthedocs.io/en/latest/lite/retro>`__ or
```
Comment (martinRenou) [inline docs/source/index.rst:8]:
```suggestion
You can give ipyleaflet a try in this documentation by clicking on `Try it on RetroLab <https://ipyleaflet.readthedocs.io/en/latest/lite/retro>`__ or
```

**Comment by jtpio** [comment]
Was the removal motivated by the potentially large initial download of assets?

**Comment by martinRenou** [comment]
I guess that was the initial motivation yes.

cc. @davidbrochart

**Pull request merged**